aries-site/src/examples/templates/dashboard/DashboardExample.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { AppIdentity } from 'aries-core';
+import { Box, Button, Header, Heading, Menu } from 'grommet';
+import { DashboardTiles } from '.';
+import { ScreenContainer, SidebarExample } from '..';
+
+export const DashboardExample = ({ mobile }) => {
+  return (
+    <ScreenContainer mobile={mobile}>
+      <Box
+        direction="row"
+        fill
+        width={mobile && { max: 'large' }}
+        height={mobile && { max: 'xlarge' }}
+        overflow="auto"
+      >
+        {!mobile && <SidebarExample mobile={mobile} />}
+        <Box pad={{ horizontal: 'medium', bottom: 'medium' }} flex>
+          <AppHeader />
+          <PageHeader />
+          <DashboardTiles mobile={mobile} />
+        </Box>
+      </Box>
+      {mobile && (
+        <SidebarExample
+          mobile={mobile}
+          direction="row"
+          fill="horizontal"
+          style={{ position: 'absolute', bottom: 0, left: 0 }}
```

Comment:
@ericsoderberghp couldn't think of a grommet-ish way to workaround this style, ideas?

Reply:
We could try having the mobile screen be:

Reply:
Good suggestions, thanks!

Reply:
I like this approach, but Layer needs to be adjusted so that you can choose what box the "position" should align width. Going to file as a feature request on Grommet.